examples/integrations/micrometer/mp/pom.xml

```diff
@@ -51,7 +51,12 @@
         </dependency>
         <dependency>
             <groupId>io.helidon.media</groupId>
-            <artifactId>helidon-media-jsonp</artifactId>
+            <artifactId>helidon-media-jsonb</artifactId>
```

Comment:
The `helidon-media-jsonb` dependency should not be needed. Can you remove it?

Reply:
Done